### PR TITLE
adds contributing md and cleans up files

### PR DESCRIPTION
**Pull request by katiedavis**
Adds contributing.md and updates readme. 
I'd love some feedback on the readme.

I also put things in alphabetical order.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+## Requirements
+
+- If you don’t have one, [create a Shopify partner account](https://partners.shopify.com/signup).
+- If you don’t have one, [create a Development store](https://help.shopify.com/en/partners/dashboard/development-stores#create-a-development-store) where you can install and test your app.
+- In the Partner dashboard, [create a new app](https://help.shopify.com/en/api/tools/partner-dashboard/your-apps#create-a-new-app). You’ll need this app’s API credentials during the setup process.
 
 ## Usage
 
-This repository is in active development and not reccomended for public usage at this time.
+This repository is intended to be scaffoled by the [Shopify-App-CLI](https://github.com/Shopify/shopify-app-cli) and includes code to scaffold code into your app.
```
[the remark this answers is not in the document]
Comment (gfscott) [reply README.md]: also `includes code to scaffold code into your app` is awkward.

Perhaps instead:
> This repository is used by [Shopify App CLI](https://github.com/Shopify/shopify-app-cli) as a scaffold for Node apps. You can clone or fork it yourself, but it’s faster and easier to use Shopify App CLI, which handles additional routine development tasks for you.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
 
 ## License
 
-The gem is available as open source under the terms of the [MIT License](https://opensource.org/licenses/MIT).
+This is available as open source under the terms of the [MIT License](https://opensource.org/licenses/MIT).
```
Comment (gfscott) [inline README.md]: I'd suggest "This respository" just for clarity

File: README.md [hunk 3]
```diff
@@ -1,17 +1,30 @@
-# Webgen Embedded app
+# Shopify App Node
+
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md)
-[![Build Status](https://travis-ci.com/Shopify/webgen-embeddedapp.svg?branch=master)](https://travis-ci.com/Shopify/webgen-embeddedapp)
+[![Build Status](https://travis-ci.com/Shopify/shopify-app-node.svg?branch=master)](https://travis-ci.com/Shopify/shopify-app-node)
 
-Boiler plate to create an embedded Shopify app made with Node, [Next.js](https://nextjs.org/), and [Shopify-koa-auth](https://github.com/Shopify/quilt/tree/master/packages/koa-shopify-auth).
+Boiler plate to create an embedded Shopify app made with Node, [Next.js](https://nextjs.org/), [Shopify-koa-auth](https://github.com/Shopify/quilt/tree/master/packages/koa-shopify-auth), [Polaris](https://github.com/Shopify/polaris-react), and [App Bridge React](https://github.com/Shopify/app-bridge/tree/master/packages/app-bridge-react).
```
Comment (gfscott) [inline README.md]: "Boilerplate" is one word!

**Comment by katiedavis** [comment]
@gfscott Thank you! Changes applied :)

**Comment by dan-gamble** [comment]
Just asking a question here so I don't open another issue!

I see this is a scaffold project for the Shopify CLI but that seems to be internal.

Is it part of the roadmap that the wider community would be able to utilise this repo?

**Comment by katiedavis** [comment]
Hey @dan-gamble . At Unite, our partner and developer conference, we'll be sharing our roadmap for developer tools. We recommend checking out unite.shopify.com next week to learn more.

**Comment by dan-gamble** [comment]
Awesome, thanks for the update @katiedavis, I'll be there :)

**Pull request merged**